src/dftbp/timedep/timeprop.F90

```diff
@@ -2234,6 +2236,14 @@ subroutine initTDOutput(this, dipoleDat, qDat, energyDat, populDat, forceDat, co
       end do
     end if
 
+    iErr = -999
+    if (this%tPump) then
+      call execute_command_line("mkdir "//trim(pumpFilesDir), exitstat=iErr)
+      if (iErr /= 0) then
+        write (stdOut,*) 'cannot create '//trim(pumpFilesDir)//', error status of mkdir: ', iErr
```

Comment:
Is this fatal? If so, call error()/propagate a status error upwards

Reply:
It's not fatal, I tried leaving the directory there and running twice, the result is the same. It's a warning. I got it from the poisson code.

Reply:
What happens if the directory is missing and can't be created?

Reply:
Then it will be caught by writeRestartFile, which has the corresponding propagate status error